modules/maven-plugin-tester/pom.xml

```diff
@@ -29,7 +29,7 @@
                     <execution>
                         <id>RunLocally</id>
                         <configuration>
-                            <runXmlProjects>false</runXmlProjects>
+                            <ignoreXmlProjects>true</ignoreXmlProjects>
```

Comment:
maybe we should call this ignoreProjectFiles instead - using "Xml" here might be confusing!?